### ISSUE TITLE
Build fails in Visual Studio on Mac using Parallels

### ISSUE THREAD

**Issue by guitarzan** [microsoft/slow-cheetah#212]
I get an error message when trying to build a project in Visual Studio on a Mac using Parallels to run Windows 10. This happens with both versions 3.2.26 and 3.2.20. I can build the same projects on a Windows computer with no problem. Also, I am able to build on the Mac if I downgrade SlowCheetah to version 3.1.66.

The error message is:

> `Could not load file or assembly 'file://\\Mac\Home\Documents\Projects\MyProject\packages\Microsoft.VisualStudio.SlowCheetah.3.2.26\build\\..\tools\Microsoft.VisualStudio.SlowCheetah.dll' or one of its dependencies. The system cannot find the file specified.`

Note the double backslash in the path at `build\\..\tools` which was added in commit d6c9cf706fa321be5e1751644ade49f2a64d8fce. Removing the extra backslash from `Microsoft.VisualStudio.SlowCheetah.targets` inside the packages folder fixes the problem.

It seems Windows (native) can ignore the double backslash, but Windows on Parallels can't? According to the [MSBuild documentation](https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-reserved-and-well-known-properties?view=vs-2019), the variable `$(MSBuildThisFileDirectory)` already includes the final backslash, so an extra backslash should not have been needed in the targets file.

**Comment by guitarzan** [comment]
More info: Parallels maps the MacOS user folder to a network drive in Windows: `\\Mac\Home\...`. When I map this to a drive letter instead (using `pushd \\Mac\Home`) and open the project from there, it works.

So the problem appears to be with network drives, not Parallels itself, but I don't have access to a Windows native computer at the moment to test.

**Comment by Yaz-X** [comment]
looks like this project is dead and no one is replying anymore.

**Comment by guitarzan** [comment]
Yeah, I noticed. It's a shame, but I figure putting this here may help someone who has the same problem. I've already had to refer back to it a couple times to remember how I fixed it, so it has at least helped me!

**Comment by Yaz-X** [comment]
i think i will implement my own transformation either using MSBuild tasks or just a simple post build copy/paste commands.

**Comment by dougwaldron** [comment]
FYI, this was fixed in #208.
